modules/using-an-algorithm/pages/index.adoc

```diff
@@ -12,11 +12,138 @@ Each algorithm has its own folder, where you can find the implementation of the
+The sub-packages correspond to a category of GDS algorithm.
+
+[source, gsql]
+----
+GSQL > SHOW PACKAGE GDBMS_ALGO
+
+Packages GDBMS_ALGO:
+  - Sub-Packages:
+    - centrality
+    - classification
+    - community
+    - graphML
+    - path
+    - similarity
+    - topological_link_prediction
+----
+
+Use dot notation to explore each sub-package.
+
+[source, gsql]
+----
+GSQL > SHOW PACKAGE GDBMS_ALGO.centrality
+
+Packages GDBMS_ALGO.centrality:
+  - Object:
+      - Packaged Queries:
+        - article_rank(string v_type, string e_type, float max_change, int maximum_iteration, float damping, int top_k, bool print_results, string result_attribute, string file_path)
+        - betweenness_cent(set<string> v_type_set, set<string> e_type_set, string reverse_e_type, int max_hops, int top_k, bool print_results, string result_attribute, string file_path, bool display_edges)
+...
+----
+
+=== Run a packaged query
+
+To run (or 'call') a packaged query, you must first be using a graph.
+Run the GSQL command `USE GRAPH <graph name>` first.
+
+Use the `CALL` command with a specific query name from a package and include the parameters.
+
+[source.wrap, gsql]
+----
+GSQL > USE GRAPH ldbc_snb
+Using graph `ldbc_snb`
+
+GSQL > CALL GDBMS_ALGO.community.louvain(["Person"],["KNOWS"],"weight",10,"","",true)
+----
+
+In this example, we call the xref:community-algorithms:louvain.adoc[] algorithm, specifying:
+
+* A set of strings `["Person"]` as the vertex types to use
+* A set of strings `["KNOWS"]` as the edge types to traverse
+* A string `"weight"` as the edge weight attribute
+* An integer `10` as the maximum limit of iterations
+* An empty string `""` for the result attribute, ensuring nothing will be written to any vertices
+* An empty string `""` for the file path, ensuring the results will not be written as a file
+* A boolean `true` to print the results to the console in JSON format.
+
+The first time you call a particular query on a graph, it goes through a short installation process.
+This only needs to happen once per query per graph.
+
+On the `ldbc_snb` graph, the results look like this after the query runs:
+
+[source.wrap, gsql]
+----
+
+------Running query------
+
+{
+  "error": false,
+  "message": "",
+  "version": {
+    "schema": 0,
+    "edition": "enterprise",
+    "api": "v2"
+  },
+  "results": [
+    {"AllVertexCount": 9163},
+    {"InitChangeCount": 0},
+    {"VertexFollowedToCommunity": 355},
+    {"VertexFollowedToVertex": 0},
+    {"VertexAssignedToItself": 729},
+    {"FinalCommunityCount": 9537}
+  ]
+}
+----
+
+xref:gui:graphstudio:overview.adoc[GraphStudio] does not support using the `CALL` command for packaged template queries.
+To install queries on GraphStudio, use the non-packaged method in the next section.
+
+== Non-Packaged queries
+
+TigerGraph provides an open-source GitHub repository with the full text of each query algorithm.
+These queries can be installed just like any other GSQL query.
+
```

Comment:
Moreover, the source code for most algorithms is included in the file system of the database.  Users can choose to install some or all of these queries with a single GraphStudio operation.  This automated installation will automatically take care of any subquery or UDF dependencies.
[I'm not sure how to write a hyperlink: https://staging--docs-tigergraph.netlify.app/gui/current/graphstudio/write-queries#_graph_data_science_library_queries]